spec/unit/postmark/mail_message_converter_spec.rb

```diff
@@ -377,13 +377,17 @@
 
   describe 'passing message stream' do
     context 'when not set' do
-      specify { expect(mail_message.to_postmark_hash.keys).not_to include('MessageStream') }
+      specify { expect(subject.new(mail_message).run).not_to include('MessageStream') }
     end
 
     context 'when set' do
-      before { mail_message.message_stream = 'weekly-newsletter' }
+      before do
+        mail_message.message_stream = 'weekly-newsletter'
+    end
 
-      specify { expect(mail_message.to_postmark_hash).to include('MessageStream' => 'weekly-newsletter') }
+      it 'adds message stream as a mail header' do
```

Comment:
```suggestion
      it 'passes message stream to the API call' do
```
or something like that. This is not a "mail header"